workbench/wsgi.py

```diff
@@ -15,14 +15,12 @@
 """
 import os
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "workbench.settings")
-
 # This application object is used by any WSGI server configured to use this
 # file. This includes Django's development server, if the WSGI_APPLICATION
 # setting points here.
-# pylint: disable=wrong-import-position
 from django.core.wsgi import get_wsgi_application
 
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "workbench.settings")
```

Comment:
I think this actually does need to come before the `get_wsgi_application` import; there's a `from django.conf import settings` in one of the indirectly imported modules.

Reply:
Ah yes, an overzealous isort.